docs/user-guide.rst

```diff
@@ -261,7 +261,8 @@ following requirements are needed:
   external subject tokens and exchange them for service account access tokens.
 - If you want to use IDMSv2, then below field needs to be added to credential_source
   section of credential configuration.
-  "aws_session_token_url": "http://169.254.169.254/latest/api/token"
+  "_imdsv2_session_token_url": "http://169.254.169.254/latest/api/token"
+  The gcloud create-cred-config command will be updated to support this soon.
```

Comment:
Should it be "imdsv2_session_token_url", no underscore?

I think the `gcloud create-....` sentence can be removed since this doc is for `google-auth` lib.

Reply:
Good catch. Yeah there should be no underscore.
Other language libraries actually have the full gcloud command in the documentation. Since python documentation doesn't have the full gcloud command, maybe we can skip this line